### ISSUE TITLE
The initializer GlorotNormal is unseeded and being called multiple times, which will return identical values each time (even if the initializer is unseeded).

### ISSUE THREAD

**Issue by danielpaques** [amaiya/ktrain#510]
Hello..

I got below warning when using ktrain. It's said that GlorotNormal is unseeded.
I know how to to seed the GlorotNormal in Keras without using ktrain, but how we do it when using ktrain?
Thank you!

`2023-09-14 14:06:40.967849: E tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:268] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2023-09-14 14:06:40.967892: I tensorflow/compiler/xla/stream_executor/cuda/cuda_diagnostics.cc:168] retrieving CUDA diagnostic information for host: ip-172-16-85-133.ap-southeast-1.compute.internal
2023-09-14 14:06:40.967900: I tensorflow/compiler/xla/stream_executor/cuda/cuda_diagnostics.cc:175] hostname: ip-172-16-85-133.ap-southeast-1.compute.internal
2023-09-14 14:06:40.968058: I tensorflow/compiler/xla/stream_executor/cuda/cuda_diagnostics.cc:199] libcuda reported version is: 525.85.12
2023-09-14 14:06:40.968080: I tensorflow/compiler/xla/stream_executor/cuda/cuda_diagnostics.cc:203] kernel reported version is: 525.85.12
2023-09-14 14:06:40.968086: I tensorflow/compiler/xla/stream_executor/cuda/cuda_diagnostics.cc:309] kernel version seems to match DSO: 525.85.12
/home/ec2-user/anaconda3/envs/python3/lib/python3.10/site-packages/keras/src/initializers/initializers.py:120: UserWarning: The initializer GlorotNormal is unseeded and being called multiple times, which will return identical values each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initializer instance more than once.
  warnings.warn(`

**Comment by amaiya** [comment]
What model are you using?

**Comment by danielpaques** [comment]
> What model are you using?

Hello @amaiya
Thanks for responding.

I'm using 'bert'.
Running this model on GCP and AWS Sagemaker's jupyter notebook, both say the same warning.

**Comment by amaiya** [comment]
Thank you - please see this issue for information on it and workarounds:

https://github.com/amaiya/ktrain/issues/488#issuecomment-1518133914
